### ISSUE TITLE
I would like to ask about variable'W'?

### ISSUE THREAD

**Issue by dazhongtu** [XU-JIA-JUN/Cable-driven-robot#2]
what does W mean?
In 'workspace_using_lmi.m' Line 27, W=[0 0 0 0 0 10000]';
In 'WORKSPACE using FM ref force.m' , W=[0 0 0 0 0 -50*9.8]';
Does it represent the gravity of end-effector?
I've read some papers, they didn't take into account gravity.I would like to ask if your program takes into account the effect of gravity? 
Looking forward to your reply

**Comment by XU-JIA-JUN** [comment]
Yes, it is the gravity vector. Acually, in workspace_using_lmi.m, you can ingore W，it does not affect final result.
However, In WORKSPACE using FM ref force.m, it does affect the final result.

**Comment by dazhongtu** [comment]
Thanks a lot  for your reply!
